src/tls.py

```diff
@@ -295,6 +436,41 @@ def set_keystore(self) -> None:
             logger.error(e.output)
             raise e
 
+    def import_cert(self, alias: str, filename: str) -> None:
```

Comment:
**todo:** Should `set_truststore` call `import_cert`? Maybe with argument `filename=ca.pem`?

Reply:
I agree, this area needs refactor, but I think it's outside of the scope of the PR.